.github/workflows/main.yaml

```diff
@@ -0,0 +1,12 @@
+name: ci
+on:
+  - push
+  - pull_request
+jobs:
+  fmt:
+    name: Format
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+    - run: go install cuelang.org/go/cmd/cue@latest
```

Comment:
This is good for now, though with CUE being pre-v1 and unstable, we may end up wanting to target more narrowly in the future. I'm happy with this at least until we see an actual problem, though.

Reply:
"if" it ever gets v1 considering how many golang projects avoid releasing v1 :P

Ideally, this dependency would be tracked somewhere in repository code to ensure users use the same cue version locally.

Reply:
yeah for sure, that's gonna be really important